librz/core/cdebug.c

```diff
@@ -1165,3 +1165,96 @@ RZ_IPI bool rz_core_debug_desc_print(RzDebug *dbg, RzCmdStateOutput *state) {
 	rz_list_free(list);
 	return true;
 }
+
+struct RzCoreDebugState {
+	RzDebug *dbg;
+	RzCmdStateOutput *state;
+};
+
+static const char *signal_option(int opt) {
+	if (opt & RZ_DBG_SIGNAL_CONT) {
+		return ("cont");
+	}
+	if (opt & RZ_DBG_SIGNAL_SKIP) {
+		return ("skip");
+	}
+	return NULL;
```

Comment:
why using a bits when you cannot have both? maybe is better to change this to an enum?